src/components/chips/js/contactChipsDirective.js

```diff
@@ -63,6 +65,8 @@ var MD_CONTACT_CHIPS_TEMPLATE = '\
               md-items="item in $mdContactChipsCtrl.queryContact($mdContactChipsCtrl.searchText)"\
               md-item-text="$mdContactChipsCtrl.itemName(item)"\
               md-no-cache="true"\
+              md-min-length="$mdContactChipsCtrl.minLength"\
+              md-has-not-found="true"\
```

Comment:
Still waiting on #9291 to be merged.

@lievenjanssen Can you ping me in a few days if it doesn't get merged so I remember to bring it back up?